src/index.ts

```diff
@@ -325,7 +492,9 @@ export default class SimpleImage {
    * @param  {object} attributes        - any attributes
    * @returns {Element}
    */
-  _make(tagName, classNames = null, attributes = {}) {
+
+
+  _make(tagName: string, classNames?: string[] | string, attributes: object = {}): HTMLElement | HTMLImageElement{
```

Comment:
can you find a way to reduce the length of this line, so it can be simple to read?